### PR TITLE
Closes #26 #27 #28 - Limit 'fluo-dev deploy' to only deploying fluo.

### LINKED ISSUE #26: Limit 'fluo-dev deploy' to only deploying fluo
Remove any initialization or starting of Fluo.  Originally from #25

### PR DESCRIPTION
**Pull request by mikewalch**
Simplify 'fluo-dev kill' command.  Combine software & data directories
into one directory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,79 +70,37 @@ Confirm that everything started by checking the monitoring pages of Hadoop & Acc
...
+cd install/fluo-1.0.0-beta-1-SNAPSHOT
 ```
 
-Deploy Fluo using the command below which will remove any existing install, rebuild fluo, install it, 
-and configure it using your configuration in `conf/fluo`:
+From here you can run the `fluo` command to administer Fluo.
 
 ```
-fluo-dev deploy
+bin/fluo status
```
Comment (keith-turner) [inline README.md]: why `bin/fluo status`?   Maybe just recommend `bin/fluo`?

Comment (mikewalch) [inline README.md]: yeah good call.. i will change.

**Comment by keith-turner** [comment]
+1 this and the apps dirs are a nice set of changes.

File: README.md (hunk 1 above)
Comment (mikewalch) [inline README.md]: fixed in 8e746d233b

**Pull request merged**